src/main/java/com/higgsup/xshop/dto/UserRegisterDTO.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2018 Azeus Systems Holdings Limited. All rights reserved.
+ *
+ * This software is the confidential and proprietary information of Azeus
+ * Systems Holdings, Ltd. ("Confidential Information"). You shall not disclose
+ * such Confidential Information and shall use it only in accordance with the
+ * terms of the license agreement you entered into with Azeus.
+ */
+
+package com.higgsup.xshop.dto;
+
+import lombok.Data;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+
+/**
+ * <b>Program ID</b>: NA <br>
+ * <br>
+ *
+ * <b>Mode</b>: <!-- Online/Batch/Library (Library by default) --> Library <br>
+ * <br>
+ *
+ * <b>Program Name</b>: NA <br>
+ * <br>
+ *
+ * <b>Description</b>: Class/Interface/Enum description here
+ * <br>
+ * <br>
+ *
+ * <b>Programming Environment</b>:
+ *
+ * <table cellspacing="0" cellpadding="0" class="PCMS2">
+ * <tr class="header">
+ * <td>Related Source</td>
+ * <td>Compiler</td>
+ * </tr>
+ * <p>
+ * <!-- Programming Environment Entries -->
+ *
+ * <tr>
+ * <td>NA</td>
+ * <td>NA</td>
+ * </tr>
+ * </table>
+ *
+ * <br>
+ *
+ * <b>File Usage</b>: <br>
+ * <p>
+ * File usage here
+ *
+ * <pre>
+ *   Enclose sample code usage in &lt;pre&gt; tags
+ * </pre>
+ *
+ * <b>Input Parameters</b>:
+ *
+ * <table cellspacing="0" cellpadding="0" class="PCMS2">
+ * <tr class="header">
+ * <td>Parameter</td>
+ * <td>Type</td>
+ * <td>Format</td>
+ * <td>Mandatory</td>
+ * <td>Description</td>
+ * </tr>
+ * <p>
+ * <!-- Input Parameter Entries -->
+ *
+ * <tr>
+ * <td>NA</td>
+ * <td>NA</td>
+ * <td>NA</td>
+ * <td>NA</td>
+ * <td>NA</td>
+ * </tr>
+ * </table>
+ *
+ * <br>
+ *
+ * <b>Screens Used</b>: NA <br>
+ * <br>
+ *
+ * <b>Processing Logic</b>: <br>
+ * <p>
+ * Place processing logic here.
+ *
+ * <br>
+ * <br>
+ *
+ * <b>External References</b>:
+ *
+ * <table cellspacing="0" cellpadding="0" class="PCMS2">
+ * <tr class="header">
+ * <td>Reference</td>
+ * <td>Description</td>
+ * </tr>
+ * <p>
+ * <!-- External References Entries -->
+ * <tr>
+ * <td>NA</td>
+ * <td>NA</td>
+ * </tr>
+ * </table>
+ *
+ * <br>
+ *
+ * <b>Program Limits</b>: NA <br>
+ *
+ * <ul>
+ * <!-- List Program Limits here using
+ * <li></li>
```

Comment:
Remove header, please. THat is very critical when you copy from another illegal resource